pgcli/main.py

```diff
@@ -156,6 +156,10 @@ def cli(database, user, host, port, prompt_passwd, never_prompt):
                     _logger.debug("status: %r", status)
                     output.extend(format_output(rows, headers, status))
                 click.echo_via_pager('\n'.join(output))
+            except KeyboardInterrupt:
+                pgexecute.reconnect()
```

Comment:
Would it be prudent to call [psycopg2.connection.cancel()](http://initd.org/psycopg/docs/connection.html#connection.cancel)? Just to make sure we're not leaving a long running query on the server and just abandoning a connection.